judge/utils/problem_data.py

```diff
@@ -10,7 +10,9 @@
 from django.utils.translation import gettext as _
 
 if os.altsep:
-    def split_path_first(path, repath=re.compile('[%s]' % re.escape(os.sep + os.altsep))):
+    repath = re.compile('[%s]' % re.escape(os.sep + os.altsep))
+
+    def split_path_first(path):
```

Comment:
This change seems semantically different to me; are you sure we don't (and won't need to) pass `repath` as a argument to this function?